assisted-events-scrape/process/host.py

```diff
@@ -0,0 +1,66 @@
+
+class Host:
```

Comment:
nit: is the class really needed? There is only one class method and the rest are all free functions.